src/elm/View/Messages.elm

```diff
@@ -100,9 +119,40 @@ viewMessage messageIndex message isLast =
     ]
 
 
-fadeInDelay : Int -> Int
-fadeInDelay messageIndex =
-    3000 * messageIndex
+fadeInDelayBasedOnMessageLength : Int -> Int -> Int -> Int
+fadeInDelayBasedOnMessageLength minChars maxChars messageLength =
+    let
+        scale min max chars =
```

Comment:
You can / should add type signatures to these inline functions - but not biggie.